Flutter/datagrid/sorting.md

```diff
@@ -347,6 +347,68 @@ Widget build(BuildContext context) {
 
 ![flutter datagrid shows sort sequence numbers during multi-column sorting](images/sorting/flutter-datagrid-showsortnumbers.gif)
 
+## Change sort number and backgroud color.
+
+The color of the sort number and its background color can be customized by using `SfDataGridThemeData.sortOrderNumberColor` and `SfDataGridThemeData.sortOrderNumberBackgroundColor` respectively.
```

Comment:
The color of the sort order number and its rounded background

Reply:
Modified